src/txadmission.cpp

```diff
@@ -1211,44 +1256,6 @@ bool ParallelAcceptToMemoryPool(Snapshot &ss,
             }
         }
 
-        {
-            READLOCK(pool.cs_txmempool);
```

Comment:
why this piece code (plus the new constraint) has been moved up just before the sigop counting mechanism?

Reply:
Two reasons...one is checkinputs is now below this section, and checkinputs should be the last thing we do since it takes the  most resources.  Second reason I reduced the scope of the READLOCK just a little.

Reply:
I see. Makes perfect sense. thanks